### PR TITLE
Feature/header and footer component

### PR DESCRIPTION
**Pull request by j8seangel**
__NOTE:__ Closed https://github.com/Vizzuality/csn-tool/pull/2 because this PR contains it (please read notes).

* Add [header](https://github.com/Vizzuality/csn-tool/pull/2) 
* Add footer component
* Refactor MainNav to use in header and footer
* Add basic layout of stay-update component.

### CODE REVIEW

**Comment by geriux** [comment]
I think the postcss-hexrgba plugin is not in the package.json because I can't install the project, can you please check?

File: app/index.html [hunk 1]
```diff
@@ -29,6 +29,20 @@
   </head>
 
   <body>
+    <div class="icons">
+      <svg style="position: absolute; width: 0; height: 0;" width="0" height="0" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <defs>
+          <symbol id="logo-small" viewBox="0 0 209 30">
+            <title>Logo small</title>
+            <g fill="none" fillRule="evenodd">
```
Comment (geriux) [inline app/index.html]: How are we going to manage the SVG icons? What about icomoon?

**Comment by j8seangel** [comment]
Fix the missing dependency, I forgot `--save` in the npm install 😕

File: app/index.html (hunk 1 above)
Comment (j8seangel) [inline app/index.html]: That sounds good, do you want to add the .json project ?

Comment (geriux) [inline app/index.html]: Alright I'll add it to PT so I don't forget

**Pull request merged**